lib/simp/rake/build/pkg.rb

```diff
@@ -452,7 +452,7 @@ def populate_rpm_dir(rpm_dir)
 
           args.with_defaults(:key => 'dev')
           args.with_defaults(:rpm_dir => File.join(File.dirname(@rpm_dir), '*RPMS'))
-          args.with_default(:force => 'false')
+          args.with_defaults(:force => 'false')
 
```

Comment:
Would be nice if you also updated the description to list the 3 optional arguments and their defaults.